doc/concepts.rst

```diff
@@ -789,6 +789,10 @@ The following constructions are available:
   ``setenv``, ``ignore-<outputs>``, ``with-stdin-from`` and
   ``with-<outputs>-to``. This action is available since Dune 2.0.
 - ``(progn <DSL>...)`` to execute several commands in sequence
+- ``(concurrent <DSL>...)``` to execute several commands concurrently.
+  **Warning:** This is limited by the number of available jobs to Dune.
+  Therefore care must be taken when writing actions that require a concurrent
+  run, such as running a server-client test with `-j 1`.
```

Comment:
```suggestion
- ``(concurrent <DSL>...)``` to execute several commands concurrently
  and aggregate all resulting errors, if any.
  **Warning:** The concurrency is limited by the `-j` flag passed to Dune.
  In particular, if Dune is running with `-j 1`, these commands will actually
  run sequentially, which may cause a deadlock if they talk to each other.
```

Reply:
(I think this is a little clearer.)